src/Listener/ResponseListener.php

```diff
@@ -17,40 +17,38 @@
 use Sonata\PageBundle\CmsManager\DecoratorStrategyInterface;
 use Sonata\PageBundle\Exception\InternalErrorException;
 use Sonata\PageBundle\Page\PageServiceManagerInterface;
-use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
 use Symfony\Component\HttpFoundation\Cookie;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Twig\Environment;
 
 /**
  * This class redirect the onCoreResponse event to the correct
  * cms manager upon user permission.
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
- *
- * @final since sonata-project/page-bundle 3.26
  */
-class ResponseListener
+final class ResponseListener
 {
     /**
      * @var CmsManagerSelectorInterface
      */
-    protected $cmsSelector;
+    private $cmsSelector;
 
     /**
      * @var PageServiceManagerInterface
      */
-    protected $pageServiceManager;
+    private $pageServiceManager;
 
     /**
      * @var DecoratorStrategyInterface
      */
-    protected $decoratorStrategy;
+    private $decoratorStrategy;
 
     /**
-     * @var EngineInterface
+     * @var Environment
      */
-    protected $templating;
+    private $templating;
```

Comment:
I think we should change all $templating for $environment or $twig ( I think I prefer twig but definitely not $templating )